addons/mrp/wizard/mrp_product_produce.py

```diff
@@ -104,3 +105,12 @@ def do_produce(self, cr, uid, ids, context=None):
         self.pool.get('mrp.production').action_produce(cr, uid, production_id,
                             data.product_qty, data.mode, data, context=context)
         return {}
+
+    def default_get(self, cr, uid, fields, context=None):
+        if context is None:
+            context = {}
+        res = super(mrp_product_produce, self).default_get(cr, uid, fields, context=context)
+        prodcution = self.pool.get('mrp.production').browse(cr, uid, context['active_id'], context=context)
```

Comment:
Strange to test only if context is empty, if there are no context, context['active_id'] is undefined and an error appear